### PR TITLE
Implement multicast and compression protocol

### PR DESCRIPTION
**Pull request by yilunzhang**
### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [x] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### CODE REVIEW

File: pb/clientmessage.proto [hunk 1]
```diff
@@ -34,6 +40,7 @@ message OutboundMessage {
   uint32 nonce = 5;
   bytes block_hash = 6;
   repeated bytes signatures = 7;
+  repeated bytes payloads = 8;
```
Comment (NoelBright) [inline pb/clientmessage.proto]: What is the difference between OutboundMessage.Payload and OutboundMessage.Payloads?

File: api/websocket/server/relay.go [hunk 2]
```diff
@@ -49,10 +49,30 @@ func (ws *WsServer) sendOutboundRelayMessage(srcAddrStrPtr *string, msg *pb.Outb
 		return
 	}
 
+	payloads := msg.Payloads
```
Comment (NoelBright) [inline api/websocket/server/relay.go]: Is "sendOutboundRelayMessage" called by client?

File: pb/clientmessage.proto (hunk 1 above)
Comment (yilunzhang) [reply pb/clientmessage.proto]: Payloads is meant to replace Payload, similar to dests vs dest

File: api/websocket/server/relay.go (hunk 2 above)
Comment (yilunzhang) [reply api/websocket/server/relay.go]: It's called in `api/websocket/server/server.go`

**Comment by NoelBright** [comment]
Looks good to me.

**Pull request merged**